printer_mods/padok/chain_anchor-ADXL345_mount/README.md

```diff
@@ -0,0 +1,50 @@
+# Chain Anchor - ADXL345 mount Mod
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/VORON2_v2.4_Assembly_-_Chain_Anchor_-_IGUS.png)
+
+Recently I started exploring the input shaper feature of klipper. Unfortunately I had to realize that it is not so easy to attach the ADXL345 in such a way that useful measurements can be collected.
+Therefore, I have created this mod, some drawings are of course still WIP but I think I have reached a stage where it is useful to get feedback.
+
+## Construction
+
+I downloaded the CAD files of the Afterburner hotend and searched for a suitable place to put the ADXL345 adapter board. I found the Chain Anchor on the backside, which offered exactly the right amount of space for my purpose.
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/VORON2_v2.4_Assembly_1.png)
+
+I printed the part on my printer using eSUN ABS+ Black/PeekGreen like all the other parts, but this one was unfortunately a little bit overextruded.
+After adding the threaded inserts, I sanded the plateau with some P180 sandpaper that I placed on a flat surface to ensure the best possible contact surface for the adapter board.
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/IMG_01.jpg)
+
+After that you can use this part as a replacement for the standard chain anchor.
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/IMG_06.jpg)
+
+## BOM
+
+| #   | Part                                               |
+| --- | -------------------------------------------------- |
+| /   | Afterburner: Chain Anchor Parts                    |
+| 4   | M3 ruthex Threaded Insert                          |
```

Comment:
What's a ruthex threaded insert?  Are these just standard inserts used everywhere else on a Voron?

Reply:
Yes, these are the same as the regular ones, I just wanted to specify the brand I used so it's clear I mean the inserts that have the 45° external corners like these:

![image](https://user-images.githubusercontent.com/32652531/127346097-538452dd-3bc1-48e7-8f1b-217120b48c70.png)

I didn't know how else to put it in short words.